src/Sulu/Bundle/ContentBundle/Resources/config/reference-store.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sulu_content.reference_store.content"
+                 class="Sulu\Bundle\ContentBundle\ReferenceStore\ReferenceStore">
+            <tag name="sulu_content.reference_store" alias="content"/>
+        </service>
+
+        <service id="sulu_content.reference_store_pool"
```

Comment:
That was only an idea 😄 But if you agree I would say we also have to move the files, otherwise the files a spread over different bundles...